rfc/cart-api-v1.md

```diff
@@ -0,0 +1,84 @@
+# Version 1
+
+| Issue | Status |
+| ----- | ------ |
+| #88   | Draft  |
+
+## Data Models
+
+### `Cart`
+| Property | Type | Description |
+| --- | --- | --- |
+| `id` | UUID-like `str` | Cart ID, auto-generated by the web API |
+| `user_id` | UUID-like `str` | The ID of the user that creates the collection |
+| `cart_name` | str | Cart Name
```

Comment:
Lets introduce the notion of a default cart whose name is null.

Reply:
What do you think about adding a new field `default` (`bool`)? In this approach,

1. any cart can be the default cart,
2. the backend will set `cart_name` to `"default"` if the user has no default cart or `"untitled"` otherwise,
3. for data inspection, the new field would make it obvious which cart is default without the knowledge on how the backend service chooses the default cart.

One downside for either approaches is that if there exists more than two "default" carts (by accident or direct data manipulation), the service will have to choose which one is the default cart?